<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startMove">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始运动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopMove">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止运动</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>通讯配置</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="baudrateBox">
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>数据位：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataBitsBox">
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="portNameBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>校验位：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="parityBox">
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>停止位：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stopBitsBox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>关闭串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
